python/lsst/pipe/tasks/extended_psf.py

```diff
@@ -554,10 +554,15 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
                 "No detector groups were provided to MeasureExtendedPsfTask; computing a single, "
                 "constant extended PSF model over all available observations."
             )
+            self.metadata["psfStarCount"] = 0
+            for brightStarStamps in bss_ref_list:
+                stamps = brightStarStamps.get()
+                self.metadata["psfStarCount"] += len(stamps)
```

Comment:
This produces a change in behavior dependent upon whether the user has requested a region-based ePSF, or a per-detector/per-region based ePSF. That might cause confusion down the line when interpreting this metadata value, requiring an `if/else` or a `try/except`, or somehow parsing the `detectors_focal_plane_regions` parameter further downstream.

My suggestion instead is to also construct a `dict` here with a single entry. The key can be something simple like `full` or `all` (or something else you feel appropriate), and the value can be this summed number. Then, downstream, we can simply iterate over all `psfStarCountDict.items()` or `psfStarCountDict.values()` to get the metadata we need. Does that sound like a good idea here?

Reply:
Yes, I agree with you and it sounds like a good idea! I made modifications based on your comment.

Reply:
Great, thanks Amir, this looks great!